docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  janusgraph:
+    image: experoinc/janusgraph:0.2.0
+    container_name: janusgraph-default
+    ports:
+      - "8182:8182"
+    volumes:
+      - "janusgraph-default-data:/var/lib/janusgraph"
```

Comment:
I think a comment would make sense here (and at the same place in the `docker-compose-mount.yml` file) to explain that this mount only makes sense when a storage like Berkeley DB is used that stores the data on disk and that runs in the same container.